src/components/scrollbar/styles/SScrollbar.tsx

```diff
@@ -0,0 +1,41 @@
+import styled, { css } from 'styled-components';
+import { MIN_SILVER, MIN_GOLD, MIN_PLATINUM, MIN_DIAMOND } from '../../../constants/styles/mediaquerys';
+
+const Bronze = css`
+width: 300px;
+height: 300px;
+background-color: #FFFFFF;
+overflow-y: scroll;
+padding: 1rem;
+::-webkit-scrollbar {
```

Comment:
@steiningermona remove all the box styling that is not needed and should be defined by the content box, not this component. Lines #5 to #9.

Reply:
You may want to keep the overflow-y, though.